keyboards/efreet/efreet.h

```diff
@@ -0,0 +1,43 @@
+/* Copyright 2019 Amber Holly
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "quantum.h"
+
+/* This a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+
+ #define LAYOUT( \
```

Comment:
You can enable [Community Layout](https://docs.qmk.fm/#/feature_layouts) support by renaming this macro to `LAYOUT_ortho_4x12` and adding a layout macro named `LAYOUT_planck_mit` that omits the position that is unused when building this with a 2u Spacebar.

```suggestion
 #define LAYOUT_ortho_4x12( \
```

Community Layout support means a user can write one keymap and use it on every keyboard they own that shares a physical layout (if the board supports the feature).